### ISSUE TITLE
Box_padding is shift on the fisrt row/column

### ISSUE THREAD

**Issue by zwwlouis** [tesseract-ocr/tesseract#1908]
Tesseract Open Source OCR Engine v4.0.0-beta.4-18-g4370
text2image 4.0.0-beta.4-18-g4370

### Current Behavior:
Since I'v found that model is sensitive on img padding, I'm trying to add some random box_padding when rendering img
But the words on the first row or column are wrong in padding, as the picture below.
![eng_box_padding](https://user-images.githubusercontent.com/4935248/45527968-d659d880-b810-11e8-9740-6b3507663eb6.jpg)

Some words are always on the first line, So they are always geting wrong box info which will likely lead to overfiting as I know.

 ### Expected Behavior:
I expect box_padding to be uniformly distribute no matter the word is on the edge or inside

**Comment by amitdo** [comment]
The bounding boxes of characters are not very reliable when the lstm engine is being used for text recognition.

Do you get the right text output?

**Comment by zwwlouis** [comment]
Nope, I use eng training text to make an example.  For chinese word it always box up character by character.
This box shift influence my model more or less. 
> The bounding boxes of characters are not very reliable when the lstm engine is being used for text 
> recognition.
> 
> Do you get the right text 
> output?
